packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -84,7 +84,11 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   /* @conditional-compile-remove(call-readiness) */
   const [audioState, setAudioState] = useState<PermissionState | 'unsupported' | undefined>(undefined);
   /* @conditional-compile-remove(call-readiness) */
-  getDevicePermissionState(setVideoState, setAudioState);
+  try {
```

Comment:
this shouldn't be necessary as getDevicePermissionState should already be catching the error:
![image](https://user-images.githubusercontent.com/2684369/208192487-e3336040-9eeb-4f5a-ae8e-ef436c2273f9.png)

Reply:
I don't think these are working because Navigator.Permissions is undefined
![image](https://user-images.githubusercontent.com/94866715/208192857-6d3332f2-b4d5-4cac-9e48-cef14a85cee7.png)
this is the error found in browserstack when this function is called.

Reply:
ah makes sense, can you move the try catch still to here so that 'unsupported' is correctly being set?

There's also a couple things that should be fixed about this function:
* Its signature reads as if it should be synchronous as it doesn't return an awaitable promise 
* it's called `getXXXXX` which implies you are trying to get the permissions not populate some side state with this function

Can you update this? Something like:
```
In fact I would expect the try catch here to fail.. I guess the permissions .query must throw before the promise is constructed,
can you update this to be an inline async function instead of using the older promise syntax.
So something like:
```ts
const queryCameraPermissionFromPermissionsAPI = () => {
  try {
     return await navigator.permissions.query({ name: 'camera' as PermissionName });
  catch () {
    return 'unsupported'
  }
}

// another function as above but for mic

export const populateDevicePermissionState = async (
  setVideoState: (state: PermissionState | 'unsupported') => void,
  setAudioState: (state: PermissionState | 'unsupported') => void
): Promise<void> => {
  const [cameraPermission, microphonePermission = await Promise.all([
    queryCameraPermissionFromPermissionsAPI,
    queryMicrophonePermissionFromPermissionsAPI
  ]);

  setAudioState(microphonePermission);
  setVideoState(cameraPermission);
};
```
(syntax could be very wrong here, just typing as an example)

Reply:
on it!